changelog.md

```diff
@@ -1,4 +1,7 @@
 # Future
+- [Fixed] `findAndCountAll` generates wrong SQL query [#5445](https://github.com/sequelize/sequelize/issues/5445)
+- [Fixed] `count` methods pollute the options.includes [#4191](https://github.com/sequelize/sequelize/issues/4191)
```

Comment:
Right, more tests are awesome! Just wanted to check if it was already broken, and doesn't seem like it was